AeroBody/Composites/index.rst

```diff
@@ -1,18 +1,29 @@
 Composites
-==============
+====
 
 Our body shell and ribbing (along with a myriad of components outside of Aerobody) are made with composites. Two materials, a matrix and its reinforcement, are combined to produce a composite material with unique and desirable properties. 
 
+	.. figure:: /Aerobody/Composites/Picture1.png
+	
 We use epoxy resin as a matrix and carbon fiber as the reinforcement to manufacture carbon fiber reinforced polymers (CFRP). We use CFRPs because of their high strength to weight ratio. This is a crucial property for the body shell, which should be strong enough to withstand any impact to the car and support the load of the chassis while adding as little weight to the car as a whole. 
 
 Manufacturing a CFRP involves saturating the carbon fiber with a thermosetting epoxy resin that will harden and bind the reinforcement fibers together. There are multiple techniques that can be used. 
 
-Using a wet layup involves using alternating layers of resin and composite fibers. Vacuum bagging uses a wet layup placed under a vacuum. The vacuum ensures that the resin is spread out and will remove resin if there is a heavy excess. Vacuum infusion uses a vacuum to draw resin from a container and pulls it through the part, with the excess being drawn out. Vacuum infusion requires the most amount of setup, but the least amount of work during the process. Generally, a vacuum infused part will be the lightest, but careful work and time can make a wet layup just as light.
+	.. figure:: /Aerobody/Composites/Picture2.png
+Using a wet layup involves using alternating layers of resin and composite fibers. 
+	
+	.. figure:: /Aerobody/Composites/Picture3.png
+
+Vacuum bagging uses a wet layup placed under a vacuum. The vacuum ensures that the resin is spread out and will remove resin if there is a heavy excess. 
+	
+	.. figure:: /Aerobody/Composites/Picture4.png
```

Comment:
All the pictures in this file aren't showing up. Looks like it's trying to pull the image from `AeroBody/Composites/Aerobody/Composites/Picture4.png` because it's being treated as a relative URL. Looks like this is a known bug for .rst files (see [issue here](https://github.com/sphinx-doc/sphinx/issues/7772)). Try changing it to be relative so that it's defined as  `.. figure:: Picture4.png`, `.. figure:: Picture3.png`, etc.